synapse/cortex.py

```diff
@@ -1206,6 +1222,120 @@ async def coreQueuePop(self, name, offs):
     async def coreQueueSize(self, name):
         return self.multiqueue.size(name)
 
+    @s_nexus.Pusher.onPushAuto('tag:model:set')
+    async def setTagModel(self, tagname, name, valu):
+        '''
+        Set a model specification property for a tag.
+
```

Comment:
I think this is the place (for lack of a better place) where you'd document what the 'regex' name means. That it is a list is important to put somewhere.